geode-core/src/main/java/org/apache/geode/internal/DistributionLocator.java

```diff
@@ -166,9 +166,9 @@ public void run() {
 
       try {
```

Comment:
There is also a static LOAD_SHARED_CONFIGURATION variable in this class that should be removed.